### ISSUE TITLE
Disabling subtitles

### ISSUE THREAD

**Issue by Sylvvvia** [kaltura/nginx-vod-module#1221]
Hello,

I'm having issue with the module that is reading subtitles from mp4 video. I would like to disable the subtitles that are automaticly being read from mp4 source video but i'm not sure how to do it. Subtitles are being read as Unkown which causes issues with other subtitles that are loaded to video externally.

**Comment by sergiyshutov** [comment]
Agree. Subtitles are automatically turned on on the TV if "vod none". How can I turn them off?

**Comment by sergiyshutov** [comment]
@erankor can you please fix this bug?
